package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wdio-lambdatest-service",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
Do not update version in package-lock manually. After updating package.json do a npm i, package-lock will be updated automatically.